src/witan/app/forecast.clj

```diff
@@ -16,15 +17,16 @@
            in_progress
            series_id
            created] :as forecast}]
-  (-> forecast
-      (dissoc :descendant_id
-              :in_progress
-              :series_id
-              :created)
-      (assoc :descendant-id descendant_id
-             :in-progress? in_progress
-             :series-id series_id
-             :created (util/java-Date-to-ISO-Date-Time created))))
+  (let [cleaned (-> forecast
+                    (dissoc :descendant_id
+                            :in_progress
+                            :series_id
+                            :created)
+                    (assoc :descendant-id descendant_id
+                           :in-progress? in_progress
+                           :series-id series_id
+                           :created (util/java-Date-to-ISO-Date-Time created)))]
```

Comment:
hmm... that would be why everything is snake_case in hecuba. Saves trouble in the json as well IIRC